v2/pkg/graphqlerrors/errors.go

```diff
@@ -49,6 +49,18 @@ func RequestErrorsFromError(err error) RequestErrors {
 	}
 }
 
+// Used by Cosmo
+func RequestErrorsFromApolloCompatibilityError(err *operationreport.ApolloCompatibilityError) RequestErrors {
```

Comment:
Looks to me that when used only in Cosmo that we can implement it in the router. All interfaces are public.

Reply:
I just put it in the same place as the original. I can move it to the router.